app/src/main/java/com/example/bluromatic/ui/theme/Theme.kt

```diff
@@ -36,12 +37,14 @@ private val DarkColorScheme = darkColorScheme(
     primary = md_theme_dark_primary,
     secondaryContainer = md_theme_dark_secondaryContainer,
     background = md_theme_dark_background,
+    surface = md_theme_dark_surface,
```

Comment:
Do we need this color?

Reply:
Refactored surface color code so I was able to remove this.